.github/actions/e2e/run-log-tests/action.yml

```diff
@@ -39,7 +39,7 @@ runs:
       with:
           name: wp(${{ env.E2E_WP_VERSION }})-wc(${{ env.E2E_WC_VERSION }})-${{ env.E2E_GROUP }}-${{ env.E2E_BRANCH }}
           path: |
-            tests/e2e/screenshots
+            screenshots
```

Comment:
We have 2 types of screenshots with E2E tests.
1. Automatic screenshots when a test fails, which is handled by the [`@woocommerce/e2e-environment`](https://www.npmjs.com/package/@woocommerce/e2e-environment#test-screenshots) library, which saves the screenshots to `./tests/e2e/screenshots`
2. Manual screenshots taken using the `takeScreenshot` helper, which saves the screenshots to `./screenshots`

Considering both the cases, I think it would be best to have both `./screenshots` & `./tests/e2e/screenshots` under GH artifact path because the contents of `./tests/e2e/screenshots` will be removed before every test suite run, which may cause losing the data inside `./tests/e2e/screenshots` if we make some changes to the test config in future.

Adding both with include files from both to the artifact zip file.

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/7955/commits/29764b49485aeb64bbfc8958196d7d8213249a13